doc/source/whatsnew/v0.20.0.txt

```diff
@@ -985,3 +985,5 @@ Bug Fixes
 - Bug in ``pd.melt()`` where passing a tuple value for ``value_vars`` caused a ``TypeError`` (:issue:`15348`)
 - Bug in ``.eval()`` which caused multiline evals to fail with local variables not on the first line (:issue:`15342`)
 - Bug in ``pd.read_msgpack`` which did not allow to load dataframe with an index of type ``CategoricalIndex`` (:issue:`15487`)
+
```

Comment:
FYI for the future if you put this somewhere *in* the Bug Fixes section , rather than the end you won't have merge conflicts. (we have blank lines for this purpose)